src/core/models/result.js

```diff
@@ -101,4 +104,28 @@ export default class Result {
      */
     this.distanceFromFilter = data.distanceFromFilter || null;
   }
+
+  /**
+   * Constructs an SDK Result from an answers-core Result
+   *
+   * @param {Result} result from answers-core
+   * @returns {@link Result}
+   */
+  static fromCore (result) {
+    const highlightedData = HighlightedFieldMap.fromCore(result.highlightedValues);
+    const details = highlightedData.description || result.description;
+    const truncatedDetails = truncate(details);
+
+    return new Result({
```

Comment:
Does the Core library's search response have a notion of `source`? The `source` could be KG, Google CSE, Algolia, etc. The `source` may impact how the SDK's `Result` is constructed. I notice I left it out of the tech design 👎 , but I think it's something we need to account for.

Reply:
The `answers-core` handles the transformations from the various sources. I think it may be worth adding `source` as a property to the core however since the properties on the rawData field will depend on the source